airbyte-webapp/src/core/jsonSchema/schemaToYup.ts

```diff
@@ -0,0 +1,102 @@
+import { JSONSchema6 } from "json-schema";
```

Comment:
That could be a small OSS project by itself :)

Reply:
There is actually one, but it doesn't support oneOf conditions :( I planned to replace custom yup schema generation with something already existent, but our validation also depends sometimes on a currently selected state in UI.

It may be worth it to create PR or just branch out from some other tool. I am considering how better approach full yup schema generation without much efforts :)